docs/tutorial/Tutorial.md

```diff
@@ -303,41 +306,76 @@ You will also have to separately rename any reference to `NoGC` to `MyGC`.
    and then type `mygc`, and repeat for `NoGC`.
 4. In order to use MyGC, you will need to make some changes to the following 
 files. 
-    1. `mmtk-core/Cargo.toml`, under `#plans`, add: 
+    1. `mmtk-core/src/plan/mod.rs`, add:
         ```rust
-        mygc = ["immortalspace", "largeobjectspace"]
+        pub mod mygc;
         ```
-        This adds a build-time flag for `mygc`, and tells the compiler that 
-        `mygc` will use an immortal space and large object space.
-    2. `mmtk-core/src/plan/mod.rs`, under the import statements, add:
+        This adds `mygc` as a module.
+    1. `mmtk-core/src/util/options.rs`, add `MyGC` to `PlanSelector`. This allows MMTk to accept `MyGC`
```

Comment:
2. ...?

Reply:
You can use 1. for enumeration in markdown. It will be rendered as the correct index. It is easier if we need to insert or reorder the list. But it makes it harder to read to the raw markdown file.